packages/firebase_performance/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.3.0
+
+* **Breaking Change** Assertion errors are no longer thrown for incorrect input for `Trace`s and
+`HttpMetric`s.
+* **Breaking Change** Removed `getAttribute()` method from `Trace` and `HttpMetric` in favor of
```

Comment:
I would suggest supporting `getAttribute`, especially if you're keeping the `put` and `remove` methods. This is consistent with other implementations such as iOS, Java, and React Native Firebase. And you won't break existing code.